docs/graphql/manual/auth/authorization/role-multiple-rules.rst

```diff
@@ -0,0 +1,118 @@
+Multiple column + row permissions for the same role
+===================================================
+
+.. contents:: Table of contents
+  :backlinks: none
+  :depth: 1
+  :local:
+
+Use case
+--------
+
+In some cases we might want to allow access to certain columns for a role only if a particular condition is met
+while allowing access to other columns based on a different condition
+i.e. have different column permissions based on different row permissions.
+
+Currently it is not possible to define multiple column + row permission rules for the same role.
+
+We can work around this limitation by using :ref:`views <custom_views>`.
+
+**Example**
+
+Let's say we have a table called ``user_info`` with columns ``(id, name, city, email, phone, address)``.
+
+We want the role ``user`` to be able to access:
+
+- the ``email``, ``phone`` and ``address`` columns only if the ``id`` column is the requesting user's id i.e. the current
+  user is the owner of the row.
+
+- the ``id``, ``name`` and ``city`` columns for all rows.
+
+We can achieve this via the following steps:
+
+Step 1: Create a view 
+---------------------
+
+:ref:`Create <create_views>` a view ``user_private`` with columns ``(user_id, email, phone, address)``:
```

Comment:
Sorry, I missed this in the last review. Can we add the link on `view` instead of `create`.

Reply:
Done